restapi/stories.go

```diff
@@ -158,20 +161,22 @@ func (api api) CreateStoriesFormData(w http.ResponseWriter, r *http.Request) ren
 	story.ImageURL = imageURL
 	story.Title = r.FormValue("title")
 	story.Content = r.FormValue("content")
-	story.CurrentCity = r.FormValue("current_city")
+	story.CurrentCity = strings.Title(strings.ToLower(r.FormValue("current_city")))
```

Comment:
I would make sure this doesn't break any existing functionality (i.e do a quick search in the repo for the `currentCity` usages). If it does then let's just do the conversion in the function call `GetLatitudeAndLongitude`

Reply:
checked our current tests and it does not break them